src/definitions/behaviors/form.js

```diff
@@ -782,7 +781,7 @@ $.fn.form = function(parameters) {
           // alias
           rules: function(field, rules) {
             if($.isArray(field)) {
-              $.each(fields, function(index, field) {
```

Comment:
This is definately a bug, not a linting improvement

Reply:
Yeah the linter trigerred an `unused variable` but it's a bug. I'll submit a PR soon.